lib/request/TingClientCollectionRequest.php

```diff
@@ -21,7 +21,15 @@ public function setAgency($agency) {
   }
 
   public function getRequest() {
-    $this->setQuery('rec.id=' . $this->id);
+    $id_array[] = $this->id;
+    if(strpos($this->id, $this->agency.'-katalog') !== FALSE) {
+      list($owner, $id) = explode(':', $this->id);
+      $this->setQuery('rec.id=870970-basis:' . $id . ' OR rec.id=' . $this->id);
```

Comment:
This needs to be commented as to why.

If the id is in the local catalog, it adds a search for basis:id? Why? And what if both exists?